### PR TITLE
Relocate createOrFindClonedNode, _uncommonedNodes from OMR

### PR DESCRIPTION
**Pull request by shivammittal99**
OMR Issue: eclipse/omr#2068
OMR PR: eclipse/omr#2269

Signed-off-by: Shivam Mittal <shivammittal99@gmail.com>

### CODE REVIEW

File: runtime/compiler/trj9/codegen/J9CodeGenerator.hpp [hunk 1]
```diff
@@ -267,6 +267,12 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGeneratorConnector
    TR_BitVector *getLiveMonitors() {return _liveMonitors;}
    TR_BitVector *setLiveMonitors(TR_BitVector *v) {return (_liveMonitors = v);}
 
+   TR::Node *createOrFindClonedNode(TR::Node *node, int32_t numChildren);
+
+protected:
+   
+   TR_HashTabInt _uncommmonedNodes;               // uncommoned nodes keyed by the original nodes
```
Comment (0xdaryl) [inline runtime/compiler/trj9/codegen/J9CodeGenerator.hpp]: Could you add a `#include "infra/HashTab.hpp"` to the top of this file please?

**Comment by 0xdaryl** [comment]
Jenkins test sanity

**Comment by 0xdaryl** [comment]
Jenkins test sanity

**Comment by 0xdaryl** [comment]
Jenkins test sanity plinux

**Comment by 0xdaryl** [comment]
@shivammittal99 , would you mind rebasing so I can launch the CI tests again?  Thanks.

**Comment by shivammittal99** [comment]
@0xdaryl rebase done.

**Comment by 0xdaryl** [comment]
Jenkins test sanity depends eclipse/omr#2269

**Comment by mstoodle** [comment]
Jenkins test sanity depends eclipse/omr#2269

**Comment by mstoodle** [comment]
Jenkins test sanity depends eclipse/omr#2269

**Comment by mstoodle** [comment]
jenkins test sanity plinux depends eclipse/omr/#2269

**Comment by mstoodle** [comment]
jenkins test sanity plinux depends eclipse/omr#2269

**Comment by mstoodle** [comment]
jenkins test sanity windows depends eclipse/omr#2269

**Comment by mstoodle** [comment]
jenkins test sanity win depends eclipse/omr#2269

**Comment by 0xdaryl** [comment]
I believe that because the CodeGenerator field was renamed this PR can go in on its own without an OMR dependence.  The corresponding OMR PR can then simply be used to remove this method and field from OMR.  @shivammittal99, can you rebase and re-push so that we can move this along?

**Comment by 0xdaryl** [comment]
Thank you for your contribution.  This PR was subsumed by #5790.  Closing.

**Pull request closed without merging**